README.md

```diff
@@ -40,42 +41,33 @@ Add the post-processor to your packer template **after** the `vagrant` post-proc
         "type":     "vagrant-s3",
         "region": "us-east-1",
         "bucket":   "my-s3-bucket",
-        "manifest": "vagrant/json/{{ user `box_name` }}.json",
-        "box_name": "{{ user `box_name` }}",
-        "box_dir":  "vagrant/boxes",
+        "manifest": "vagrant/json/{{ user `box_organization` }}/{{ user `box_name` }}.json",
+        "box_dir":  "vagrant/boxes/{{ user `box_organization` }}/{{ user `box_name` }}",
+        "box_name": "{{ user `box_organization` }}{{ user `box_name` }}",
         "version":  "{{ user `version` }}"
       }
     ]
   ]
 }
 ```
-**NOTE:** The post-processors must be a **nested array** (i.e.: a Packer sequence definition) so that they run in order. See the [Packer template documentation](http://www.packer.io/docs/templates/post-processors.html) for more information.
```

Comment:
Let me know if you want me to undrop this. I figured that if someone is so familiar with Packer that they've started hunting for custom post-processors they surely know the syntax too.